README.md

```diff
@@ -3746,12 +3746,6 @@ Other Style Guides
 ## Performance
 
   - [On Layout & Web Performance](https://www.kellegous.com/j/2013/01/26/layout-performance/)
-  - [String vs Array Concat](https://jsperf.com/string-vs-array-concat/2)
```

Comment:
this should be https://web.archive.org/web/20200414200857/https://jsperf.com/string-vs-array-concat/2